src/resultsdisassemblypage.h

```diff
@@ -56,6 +56,13 @@ struct DisassemblyOutput
     QString errorMessage;
     explicit operator bool() const { return errorMessage.isEmpty(); }
     static DisassemblyOutput fromProcess(const QString &processName, const QStringList &arguments);
+
+    struct DisassemblyLine
+    {
+        quint64 addr = 0;
+        QString disassembly;
+    };
+    QVector<DisassemblyLine> disassemblyLines;
```

Comment:
use this to replace the `QByteArray output`

Reply:
What do you mean? I need them both - QByteArray output and QVector<DisassemblyLine> disassemblyLines.